example_notebooks/moving_pictures/moving_pictures.ipynb

```diff
@@ -0,0 +1,413 @@
+    "\n",
+    "![](images/qurro3dots.png)\n",
+    "\n",
+    "As we expected, this log-ratio clearly separates `gut` samples from most of the other samples in this dataset. There are two `right palm` samples that still cluster with these `gut` samples, though, which is interesting! As an exercise for the reader -- looking at the biplot, does these two `right palm` samples having such a low log-ratio here make sense to you? Try checking out sample IDs by hovering over points in the Qurro sample plot, or by double-clicking on a sample in Emperor."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.2. Moving from autoselection to taxonomy-based filtering\n",
+    "\n",
+    "Although having a log-ratio that kind of differentiates poop samples from skin/oral samples is somewhat interesting, we might want to know more about the specific features that influence this separation. For example, what are the  assigned taxonomic classifications of the highest- and lowest- ranked features in the Axis 1 feature loadings?\n",
+    "\n",
+    "To investigate this, let's look at the tables of selected numerator / denominator features in the bottom left quadrant of the screen. These are interactive tables showing all of the features present in the currently selected log-ratio: each row represents a selected feature. You can sort the rows of these tables by any of the columns in the table.\n",
+    "\n",
+    "Let's use the `Axis 1` column to sort the numerator and denominator tables:\n",
+    "\n",
+    "![](images/qurro4.png)\n",
+    "\n",
+    "(Notice how the arrow icon for the `Numerator Features` table is pointing down, indicating that it's sorted in descending order, and how the arrow icon for the `Denominator Features` table is pointing up -- indicating that it's sorted in ascending order.)\n",
+    "\n",
+    "Once we've got things sorted, we can scroll over and check out the assigned taxonomic classifications (`Taxon`) for these features. As you scroll through the table, do you notice any patterns?\n",
+    "\n",
+    "![](images/qurro5.png)\n",
+    "\n",
+    "We can see that the highest-ranked feature for the `Axis 1` loadings is classified in the genus `Pseudomonas`, and the lowest-ranked feature is classified as `mitochondria`. Since `mitochondria` are usually  Let's try looking at the log-ratio between these features' families: so, between the features classified as being in `f__Pseudomonadaceae` and the features classified as being in `f__mitochondria`. (Note: in practice, you may want to filter `mitochondria` out of your datasets entirely -- see [this discussion](https://forum.qiime2.org/t/mitochondrial-16s-handling/1384). Feel free to try out the second lowest-ranked feature, classified as `g__Bacteroides`, as the denominator here instead!)\n",
```

Comment:
"Since `mitochondria are usually" seems to be incomplete.

Reply:
Good catch! I was trying out like 3 different ways to say "mitochondria might be garbage" but wasn't sure how to phrase it best, and I accidentally left one of the remnants in here...